packages/wallet/src/base-unlocked-wallet.ts

```diff
@@ -56,6 +56,7 @@ export class BaseWalletUnlocked extends Account {
    * @param transactionRequestLike - TransactionRequestLike
    * @returns string - Signature a ECDSA 64 bytes
    */
+  // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
On this method, I see you preferred to disable the rule instead of using a `Promise.resolve()` approach like on the `signMessage` method (for which I also asked other things).

I'm just trying to understand the differences between all these different approaches to see if we can hopefully standardize all occurrences.

Reply:
I see. If we decide to leave all of them as `async`, we can then agree on only one way to bypass this rule for all these cases.